spec/jobs/extend_petition_deadlines_job_spec.rb

```diff
@@ -23,6 +23,8 @@
     end
 
     it "increments the closed_at attribute by 1 day" do
+      petition.update(closed_at: Time.current)
+
```

Comment:
We should probably remove this job and spec as well since the concept of deadline doesn't exist